aiida/restapi/translator/data/structure.py

```diff
@@ -83,7 +83,7 @@ def __init__(self, **kwargs):
     def get_visualization_data(node, format=None, supercell_factors=[1, 1, 1]):
         """
         Returns: data in specified format. If format is not specified returns data
-        in a format required by chemdoodle to visualize a structure.
+        in xsf format in order to visualize the structure with JSmol.
         """
         response = {}
         response["str_viz_info"] = {}
```

Comment:
Here below, I would do (if we want to have 'xsf' as default):
```python
if format is None:
    format = 'xsf' 
```
and then let the logic take care of the default.
Then, in the else, raise an exception (now instead it returns 'xsf' even if the string is unknown).